splunk/packages/edit-table/src/components/KVStoreUploader.jsx

```diff
@@ -27,30 +28,66 @@ export default function KVStoreUploader({
     setUploadModalOpen,
     collectionName,
     splunkApp,
+    kvStore,
     tableMetadata,
     setInfoMessage,
     refreshVisualization,
 }) {
     const [uploading, setUploading] = useState(false);
     const [uploadedCSVContent, setUploadedCSVContent] = useState();
 
+    const executeJob = async (job, errorMessage) => {
```

Comment:
Don't we have a file do deal with the data handling? Why not move these functions there then?

Reply:
should be '../data'